Makefile

```diff
@@ -47,6 +47,7 @@ unit-tests:
 	@bash ./scripts/unittests.sh
 
 # test runs ALL the test suites.
-test: systemtests unit-tests
+test:
+	@bash ./scripts/tests.sh
```

Comment:
Makes sense. then in that case, can we change this target to `ci` or something relevant